Makefile

```diff
@@ -0,0 +1,16 @@
+GO_SOURCES = $(shell find pkg -type f -name '*.go')
+
+vendor: glide.lock
+	glide install -v --force
+
+glide.lock: glide.yaml
+	glide up -v --force
+
+test: $(GO_SOURCES) vendor
+	kafka-topics --delete --zookeeper localhost:2181 --topic test-topic
+	kafka-topics --create --zookeeper localhost:2181 --topic test-topic --partitions 1 --replication-factor 1
+	KAFKA_BROKERS=localhost:9092 go test -v ./...
+
+gen-mocks: $(GO_SOURCES)
```

Comment:
Since the dependent file is local (ie not in vendor), this should be properly listed as a `make` dependency (in addition to `gen-mocks` if we want to). 
`test` and `gen-mocks` are phony targets.

Would be nice to also have:
- a build target (see kubernetes-crds for an approach to building a lib only project) to make sure that everything compiles
- a check (using git status?) that generated mocks are in synch with sources

Reply:
@ericbottard Thanks. All fixed, including nice to haves.